smallrye-reactive-messaging-rabbitmq/src/main/java/io/smallrye/reactive/messaging/rabbitmq/OutgoingRabbitMQMetadata.java

```diff
@@ -313,4 +318,34 @@ public OutgoingRabbitMQMetadata build() {
             return metadata;
         }
     }
+
+    /**
+     * Converts this OutgoingRabbitMQMetadata to an IncomingRabbitMQMetadata.
+     * This is mainly intended for use in unit tests that relies on incoming metadata.
+     *
+     * @param exchange the exchange
+     * @param isRedeliver if it was a redelivery
+     * @return this OutgoingRabbitMQMetadata converted to an IncomingRabbitMQMetadata
+     */
+    public IncomingRabbitMQMetadata toIncomingMetadata(String exchange, boolean isRedeliver) {
+        BasicProperties basicProperties = new AMQP.BasicProperties.Builder()
+                .userId(userId)
+                .appId(appId)
+                .headers(headers)
+                .contentType(contentType)
+                .contentEncoding(contentEncoding)
+                .correlationId(correlationId)
+                .deliveryMode(deliveryMode)
+                .expiration(expiration)
+                .priority(priority)
+                .messageId(messageId)
+                .replyTo(replyTo)
+                .timestamp(Date.from(timestamp.toInstant()))
+                .type(type)
+                .build();
+        Envelope envelope = new Envelope(1, isRedeliver, exchange, routingKey);
```

Comment:
For my own understanding, why 1?

Reply:
I should have put a comment about it, but the field is never used in the in the `IncomingRabbitMQMessage` so it does not really matter, I think I may have copied the values from some other code I found but `0`might be a better value.

Reply:
I have force pushed an update that used `0` instead of `1` with a comment on why it is hardcoded.